pkg/csi/service/vanilla/controller_helper.go

```diff
@@ -151,6 +151,7 @@ func getBlockVolumeIDToNodeUUIDMap(ctx context.Context, c *controller,
 	var err error
 
 	log := logger.GetLogger(ctx)
+	log.Debugf("getBlockVolumeIDToNodeUUIDMap called for Node VMs: %+v", allnodeVMs)
```

Comment:
We don't need function name again in the debug log. This could just be: "All node VMs"

Reply:
just printing it to help debug.